massa-module-cache/src/controller.rs

```diff
@@ -43,7 +43,7 @@ impl ModuleCache {
     fn compile_cached(&mut self, bytecode: &[u8], hash: Hash) -> ModuleInfo {
         match RuntimeModule::new(
             bytecode,
-            self.cfg.compilation_gas,
+            self.cfg.gas_costs.max_instance_cost,
```

Comment:
this parameter is part of `gas_costs` and is always passed along with it up to its point of use.
Do we want to drop it?